include/capstone.h

```diff
@@ -27,19 +27,23 @@ extern "C" {
 #pragma warning(disable:4201)
 #pragma warning(disable:4100)
 #define CAPSTONE_API __cdecl
+#ifndef CAPSTONE_EXPORT
```

Comment:
It is pretty obvious I thought, when you export some API, it is reasonable to allow custom rules for your symbols. Some people may not want to pollute the namespace and could put an `__abi_tag__` for example. Whatever it is, changed since I don't need it.